2/Dockerfile.rhel7

```diff
@@ -56,6 +56,11 @@ ADD release.version /tmp/release.version
 # and its presence in the rhel image is confusing.
 RUN rm /opt/openshift/base-plugins.txt && \ 
     mkdir -p /opt/openshift/plugins && \
+    # we symlink the rpm installed plugins from /usr/lib/jenkins to /opt/openshift/plugins so that
+    # future upgrades of the image and their RPM install automatically get picked by jenkins;
+    # we use synlinks vs. actual files to delineate whether the user has overriden a plugin (and
+    # by extension taken over its future maintenance)
+    for FILENAME in /usr/lib/jenkins/*hpi ; do ln -s $FILENAME /opt/openshift/plugins/`basename $FILENAME .hpi`.jpi; done &&\
```

Comment:
does this mean we can go back and remove the logic in "run" that copies from /usr/lib/jenkins?  (https://github.com/openshift/jenkins/blob/master/2/contrib/s2i/run#L229)

Reply:
the comment in `run` should be adjusted, but we still need that code, since a release upgrade could introduce new plugins and the PV needs to be updated ...the links here only account for the initial image start

Reply:
hm.  So this logic symlinks from /usr/lib/jenkins to /opt/openshift/plugins so that the install_plugins.sh script (as executed in assemble) sees them?

and then the run script symlinks them from /usr/lib/jenkins to /var/lib/jenkins to Jenkins itself sees them?

Reply:
yes and yes

and we tolerate the additional hop with rhel to facilitate a common code path with both centos and rhel for install_plugins.sh (why /usr is not used for centos was a decision that predated my forays into this stuff, or there is an obvious explanation as to why that I'm unaware of)

Reply:
duh .... it was file permissions :-) ... the non-root user can modify /opt based on the fix permission stuff 

now, we've since modified permissions in /usr for the 32 bit vs. 64 bit stuff; so if we wanted, we could switch gears here and just use /usr

but not this PR :-)